stacks-common/src/util/secp256k1.rs

```diff
@@ -118,6 +118,7 @@ impl Default for Secp256k1PublicKey {
     }
 }
 
+/*
```

Comment:
Same.  Unless you really want to use byte arrays explicitly?  The current `serde` impl in `p256k1` will give you bytes when you need them and fast math objects otherwise.